src/web/src/app/space/model.ts

```diff
@@ -1,9 +1,46 @@
-export class Space {
+export interface SpaceListResponse {
+	items: Space[];
+}
+
+export interface Space {
 	id: string;
 	name: string;
-	title: string;
+	key: string;
+	requestsAllowed: boolean;
+	createdAt: Date;
 }
 
-export interface SpaceListResponse {
-	items: Space[];
+export interface SpaceInvitation {
+	id: number,
+	spaceId: string;
+	creatorId: string;
+	userId: string;
+	role: string;
+	createdAt: Date;
 }
+
+export interface SpaceLink {
+	id: string;
+	spaceId: string;
+	creatorId: string;
+	name: string;
+	createdAt: Date;
+	expiredAt: Date;
+}
+
+export interface SpaceUser {
+	spaceId: string;
+	userId: string;
+	isOwner: boolean;
+	joinedAt: Date;
+	role: string;
+}
+
+export type SpaceInvitationFields = {
+	spaceName: string;
+	userLogin: string;
+}
+
+export type SpaceLinkFields = {
+	spaceName: string;
+}
```

Comment:
Вместо userLogin предлагаю использовать тип UserFields
```suggestion
export type SpaceFields = {
	spaceName: string;
}
```

Reply:
В этом случае для замены SpaceInvitationFields нужно будет унаследоваться одновременно от SpaceFields и UserFields (при этом из UserFields нужно будет взять только Login)?

Reply:
Норм. К тому же эти поля возможно когда то в будущем выведем в подсказке при наведении.

Reply:
Готово.